projects/ngrx.io/content/guide/data/extension-points.md

```diff
@@ -167,6 +167,96 @@ export const selectedCatsWithOwners = createSelector(
 
 ### Replace the HttpUrlGenerator
 
+This example replaces the `DefaultHttpUrlGenerator` with a customized `HttpUrlGenerator` that pluralizes both collection resource and entity resource URLs.
+
+The implementation simply overrides `DefaultHttpUrlGenerator.getResourceUrls(string, string)`:
+
+```ts
+import {
+  DefaultHttpUrlGenerator,
+  HttpResourceUrls,
+  normalizeRoot,
+  Pluralizer
+} from '@ngrx/data';
+
+export class PluralHttpUrlGenerator extends DefaultHttpUrlGenerator {
+  constructor(private pluralizzer: Pluralizer) {
+    super(pluralizzer);
+  }
+
+  protected getResourceUrls(
+    entityName: string,
+    root: string
+  ): HttpResourceUrls {
+    let resourceUrls = this.knownHttpResourceUrls[entityName];
+    if (!resourceUrls) {
+      const nRoot = normalizeRoot(root);
+      const url = `${nRoot}/${this.pluralizzer.pluralize(
+        entityName
+      )}/`.toLowerCase();
+      resourceUrls = {
+        entityResourceUrl: url,
+        collectionResourceUrl: url
+      };
+      this.registerHttpResourceUrls({ [entityName]: resourceUrls });
+    }
+    return resourceUrls;
+  }
+}
+```
+
+In order to hook it into ngrx data we need to provide it in the module where ngrx data is imported:
```

Comment:
```suggestion
Override the `HttpUrlGenerator` provider in the root `AppModule` where `EntityDataModule.forRoot()` is imported:
```

Reply:
LGTM, but I'm not sure if I need to update the branch. Or will they come through automatically, when the change request is resolved?

Reply:
If you accept the suggestions, it will update the branch

Reply:
If you accept the suggestions, it will update the branch